agentGroup.yaml

```diff
@@ -1,8 +1,7 @@
 kind: AgentGroup
-description: Grouping of 3 agents.
+description: This agent group will run various Nuclei templates on different open ports
```

Comment:
Not unique to Nuclei. Please update the description to only mention finding exploitable vulnz.